### ISSUE TITLE
Layout issue with status bar

### ISSUE THREAD

**Issue by carsten19** [mapsplugin/cordova-plugin-googlemaps#242]
With the new version of your plugin I have a layout problem together with the IOS status bar on IOS7 and IOS8. I am using the cordova status bar plugin to disable the overlay. But that causes the map to push the whole content down by 20px.
This is how it looks like (white extra bar on top of the app and footer not completely visible).
![ios simulator screen shot 08 10 2014 17 38 18](https://cloud.githubusercontent.com/assets/839480/4562032/81948132-4f01-11e4-900e-428c98b5cad9.png)

If I hide the status bar when the device is ready and then show it again when the map plugin has initialized it works:
![ios simulator screen shot 08 10 2014 17 35 14](https://cloud.githubusercontent.com/assets/839480/4562051/a42b8f24-4f01-11e4-8c65-152ab91ea45f.png)

This effect does not happen with the older versions of your plugin and it does also not happen when no map is loaded.
I would like to get rid of this hiding and showing the status bar as it is an ugly workaround.
To me it looks like the plugin does not recognize the layout change correctly. I have also tried a forced refresh but that did not change anything.

**Comment by aramgev** [comment]
I call : 
  StatusBar.hide();
  StatusBar.show();

But it doesn't solve the issue.

**Comment by pmwisdom** [comment]
StatusBar.hide();
StatusBar.show();

Also doesn't work for me. I'm using the plugin.

**Comment by mabdelfattah** [comment]
+1

**Comment by javierClavei** [comment]
+1

**Comment by ricardouffcomp** [comment]
Finally I found a workaround to fix this issue. I only tested it in IOS 9. Let me know if it works in Android and other version of IOS:

In the "deviceready" I added the following code:

        StatusBar.hide();

        setTimeout(function () 
        {
            StatusBar.show();
        }, 1500);

And before opening the map:

//The Status Bar closes and opens quickly and it was unnoticeable in my phone
StatusBar.hide(); 
map.showDialog();
StatusBar.show();

I hope it helps.
